mtp_api/apps/core/filters.py

```diff
@@ -39,6 +39,23 @@ def filter(self, qs, value):
             return self.get_method(qs)(q).distinct()
         return self.get_method(qs)(q)
 
+    def label():
+        def fget(self):
+            if self._label is None and hasattr(self, 'parent'):
+                model = self.parent._meta.model
+                fields = []
+                for name in self.name:
+                    fields.append(django_filters.utils.label_for_filter(
+                        model, name, self.lookup_expr, self.exclude
+                    ))
+                self._label = ', '.join(fields)
+            return self._label
+
+        def fset(self, value):
+            self._label = value
+
+        return locals()
+
```

Comment:
i don't django filters do this weird thing with property and locals – is it a backwards compatibility thing for python 2.0? is this not equivalent to

```python
    @property
    def label(self):
        if self._label is None and hasattr(self, 'parent'):
            model = self.parent._meta.model
            fields = []
            for name in self.name:
                fields.append(django_filters.utils.label_for_filter(
                    model, name, self.lookup_expr, self.exclude
                ))
            self._label = ', '.join(fields)
        return self._label

    @label.setter
    def label(self, value):
        self._label = value
```